Templates/MonoGame.Templates.CSharp/content/MonoGame.StartKit/___SafeGameName___.Android/___SafeGameName___.Android.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0-android</TargetFramework>
+    <SupportedOSPlatformVersion>23</SupportedOSPlatformVersion>
+    <OutputType>Exe</OutputType>
+    <ApplicationId>com.companyname.___SafeGameName___.Android</ApplicationId>
```

Comment:
probably don't need the .Android at the end here (or other platforms).

Reply:
Then how do you tell the difference between projects when you open them all in one solution.
Or even side by side?
Unless I'm not getting what you are getting at?

Reply:
Oh I see, just in the namespaces. In that case I agree 😂

Reply:
Generally you should avoid using `Android` in anthing that might end up in a `namespace` as well. Since it can cause conflicts with the root `Android` namespace.

Reply:
But in this case its the ApplicaitionId which will end up as the apk/aab name and in the store. So on the Play store we don't need to have the name of the app has Android at the end. com.company.MyGame is better than com.company.MyGame.Android.  its more consistent with other platforms.

Reply:
@dellis1972 I need to double check, but think we need a way to separate out each platform.
I'll remove the platform from each app equivalent Id.

Reply:
We used to use `Droid` for project names, rather than Android. It might have been fixed. 
But the ApplicationId's definately do not need to have platform specific bits in them.

Reply:
@CartBlanche did you see the above comment about having `Android`at the end of the namespace?

Reply:
@dellis1972 I had seen your comment, but I seem to have lost the changes back then. Fixed in latest commit.